Tasks/ServiceFabricDeploy/task.json

```diff
@@ -15,7 +15,7 @@
         "DeploymentGroup"
     ],
     "version": {
-        "Major": 1,
+        "Major": 2,
```

Comment:
Don't update major version. We are not breaking existing functionality- except for a very corner case.